src/modules/reporting/reducers/account-disputes-state.js

```diff
@@ -0,0 +1,28 @@
+import {
+  REMOVE_ACCOUNT_DISPUTE,
+  UPDATE_ACCOUNT_DISPUTE,
+  CLEAR_ACCOUNT_DISPUTES,
+} from 'modules/reporting/actions/update-account-disputes'
+import { RESET_STATE } from 'modules/app/actions/reset-state'
+
+export default function (accountDisputes = {}, action) {
```

Comment:
Add test coverage

Reply:
test coverage for default?  test/reporting/reducers/account-disputes-test.js did I miss a case?